apps/src/applab/applab.js

```diff
@@ -1779,7 +1780,11 @@ function updateTurtleImage(turtleImage) {
   }
   turtleImage.style.left = (Applab.turtle.x + TURTLE_X_OFFSET) + 'px';
   turtleImage.style.top = (Applab.turtle.y + TURTLE_Y_OFFSET) + 'px';
-  turtleImage.style.transform = 'rotate(' + Applab.turtle.heading + 'deg)';
+  var heading = Applab.turtle.heading + TURTLE_ROTATION_OFFSET;
+  var transform = 'rotate(' + heading + 'deg)';
+  turtleImage.style.transform = transform;
+  turtleImage.style.msTransform = transform;
+  turtleImage.style.webkitTransform = transform;
```

Comment:
What's the reason for having the turtle be an img element on top of the canvas instead of drawing the image onto the canvas?